### PR TITLE
#145 first shot at SVG export

### PR DESCRIPTION
**Pull request by JanKoehnlein**

### CODE REVIEW

File: client/package.json [hunk 1]
```diff
@@ -51,13 +51,15 @@
...
   "dependencies": {
+    "file-saver": "^1.3.3",
     "inversify": "^3.1.0",
     "snabbdom": "^0.6.4",
     "snabbdom-jsx": "^0.3.1",
     "snabbdom-virtualize": "^0.6.0"
   },
   "devDependencies": {
     "@types/chai": "^3.4.35",
+    "@types/file-saver": "https://registry.npmjs.org/@types/file-saver/-/file-saver-0.0.1.tgz",
```
Comment (spoenemann) [inline client/package.json]: Why this url dependency?

Comment (JanKoehnlein) [inline client/package.json]: This is what `npm install @types/file-saver --save-dev` produced. No idea what kind of problem that is.

File: client/src/features/export/export.ts [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+export class RequestExportSvgAction implements Action {
+    kind = ExportSvgCommand.KIND
+}
+
+export class ExportSvgCommand extends HiddenCommand {
+    static KIND = 'requestExportSvg'
+
+    execute(context: CommandExecutionContext): SModelRoot {
+        if (isExportable(context.root)) {
+            const root = context.modelFactory.createRoot(context.modelFactory.createSchema(context.root))
```
Comment (spoenemann) [inline client/src/features/export/export.ts]: Is it necessary to make a schema before invoking the factory?

Comment (JanKoehnlein) [inline client/src/features/export/export.ts]: I wanted to be sure to create a clone.

**Pull request merged**